### PR TITLE
Make autoscaler CPU limit configurable

### PR DESCRIPTION
**Pull request by mweinberg2718**
Updates README, Flask configmap, and values.yaml to accommodate new CPU threshold configuration for autoscaling.

### CODE REVIEW

File: servicex/values.yaml [hunk 1]
```diff
@@ -78,6 +78,8 @@ codeGen:
 transformer:
   pullPolicy: IfNotPresent
   autoscalerEnabled: false
+  cpuLimit: 1
+  cpuScaleThreshold: 1
```
Comment (BenGalewsky) [inline servicex/values.yaml:82]: Is this a good default value for people to use?

Comment (mweinberg2718) [reply servicex/values.yaml:82]: Maybe not for the cpuScaleThreshold. After some testing on high-efficiency transformers, I would suggest that 70% provides a reasonable balance between speed of spinning up a new pod and the issue of making too many unused pods.

File: README.md [hunk 2]
```diff
@@ -224,6 +224,8 @@ or [mino](https://github.com/helm/charts/tree/master/stable/minio#configuration)
 | `minio.ingress.hosts`                | List of hosts to associate with ingress controller | nil |
 | `transformer.pullPolicy`             | Pull policy for transformer pods (Image name specified in REST Request) | IfNotPresent |
 | `transformer.autoscalerEnabled`      | Set to True to enable the pod horizontal autoscaler for transformers |  False          |
+| `transformer.cpuLimit`               | Set CPU resource limit for pod in number of cores | 1 |
+| `transformer.cpuScaleThreshold`      | Set CPU percentage threshold for pod scaling | 1 |
```
Comment (BenGalewsky) [inline README.md:228]: Please fix this to match `values.yaml`

Comment (mweinberg2718) [reply README.md]: Good catch!

**Pull request merged**